Emotion/Windows/ClientLibrary/Contract/Scores.cs

```diff
@@ -75,6 +75,26 @@ public class Scores
         /// 
         /// </summary>
         public float Surprise { get; set; }
+        
```

Comment:
Hi @ccasbre27 - while getting the top emotion is a helpful method, I think ranked scores might be of more general utility.  Something like the following:

```
        /// <summary>
        /// Create a sorted key-value pair of emotions and the corresponding scores, sorted from highest score on down.
        /// To make the ordering stable, the score is the primary key, and the name is the secondary key.
        /// </summary>
        public IEnumerable<KeyValuePair<string, float>> ToRankedList()
        {
            return new Dictionary<string, float>()
            {
                { "Anger", Anger },
                { "Contempt", Contempt },
                { "Disgust", Disgust },
                { "Fear", Fear },
                { "Happiness", Happiness },
                { "Neutral", Neutral },
                { "Sadness", Sadness },
                { "Surprise", Surprise }
            }
            .OrderByDescending(kv => kv.Value)
            .ThenBy(kv => kv.Key)
            .ToList();
        }
```
If you're interested in the name of the strongest emotion, you'll use `scores.ToRankedList().First().Key`

What do you think?